media-sound/rcenter/files/rcenter-0.6.2-pthread-pointer-type.patch

```diff
@@ -0,0 +1,14 @@
+Give a type and a name to a function parameter that will continue to be ignored
+due to C23
+https://bugs.gentoo.org/944365
+--- a/lowlevel.c
++++ b/lowlevel.c
+@@ -44,7 +44,7 @@
+ static int timerpipe[2];
+ static int quitpipe[2];
+ 
+-static void *timerloop()
++static void *timerloop(void * nothing)
```

Comment:
What is it being called with? (Why is it fine to ignore it?)

Reply:
It's called in `pthread_create(&timer_thread, NULL, timerloop, NULL);`
so, timerloop(NULL), with argument never checked.